plugin-energy-attribution/src/transform.rs

```diff
@@ -13,15 +12,16 @@ use alumet::{
     resources::Resource,
 };
 
+use crate::Metrics;
+
 pub struct EnergyAttributionTransform {
-    pub metrics: Arc<Mutex<super::Metrics>>,
+    pub metrics: super::Metrics,
     buffer_pod: HashMap<u64, Vec<MeasurementPoint>>,
     buffer_rapl: HashMap<u64, MeasurementPoint>,
 }
-
 impl EnergyAttributionTransform {
     /// Instantiates a new EnergyAttributionTransform with its private fields initialized.
-    pub fn new(metrics: Arc<Mutex<super::Metrics>>) -> Self {
+    pub fn new(metrics: Metrics) -> Self {
```

Comment:
```suggestion
    pub fn new(metrics: super::Metrics) -> Self {
```
Choose between importing `super::Metrics` or using its full path (line 18 uses `super::Metrics`, line 24 uses `Metrics`)

Reply:
done in the last commit and import (metrics structure) deleted